samples/msal-browser-samples/vue3-sample-app/src/authConfig.ts

```diff
@@ -0,0 +1,52 @@
+import { LogLevel, PublicClientApplication } from '@azure/msal-browser';
+
+// Config object to be passed to Msal on creation
+export const msalConfig = {
+  auth: {
+    clientId: '3fba556e-5d4a-48e3-8e1a-fd57c12cb82e',
```

Comment:
Do you want to remove the clientId and change it to add your clientId here?  Likewise, do we want to point to a non-PPE authority on the line just below?

Reply:
We usually leave these in on our dev samples, mostly for testing. Given that we don't have tests for this at the moment, I've removed this for now.